lib/metrics.js

```diff
@@ -1,5 +1,5 @@
 var Statsd = require('strong-statsd');
-var agent = require('./agent');
+// var agent = require('./agent');
```

Comment:
I'm ok with this, though commenting out the call to metrics.js might be shorter.